examples/with-typescript-graphql/lib/resolvers.ts

```diff
@@ -1,4 +1,4 @@
-import { QueryResolvers, MutationResolvers } from './type-defs.graphqls'
+import { QueryResolvers, MutationResolvers } from '@graphql-types@'
```

Comment:
Is the @ at the end here intended?

Reply:
Thank you for your prompt reply.
There is no particular intention. I added it because @ was attached to the example code of the migration guide of v0.18.0 of graphql-let.
In my opinion, it may or may not be present.
document -> https://github.com/piglovesyou/graphql-let/releases/tag/v0.18.0
<img width="762" alt="スクリーンショット 2021-12-21 3 10 34" src="https://user-images.githubusercontent.com/52757302/146813199-0d3afdc7-7f4c-4916-8a7a-e458f9c9a37c.png">

Reply:
then do we need `./type-defs.graphqls`file?

Reply:
Thank you for your time and reply.
Yes,` type-defs.graphqls` contains a list of Mutation operations and a list of Query operations.
Graphql needs to declare the list of mutation and query operations by writing "type Mutation(or Query)" respectively. This file is the core of graphql, so it cannot be deleted.